firmware/Core/Src/littlefs/flash_driver.c

```diff
@@ -506,8 +509,18 @@ FLASH_error_enum_t FLASH_is_reachable(SPI_HandleTypeDef *hspi, uint8_t chip_numb
         are_bytes_correct = 0;
     }
 
+    // Extract memory capacity
+    uint32_t capacity_mib = (rx_buffer[2] * 512) / 32;  // Convert to Mebibytes (MiB)
+
+    // String to print capacity in MiB
+    char capacity_str[30];
+    snprintf(capacity_str, sizeof(capacity_str), "Capacity: %lu MiB", capacity_mib);
+
+    // Print out response and capacity in human-readable format MiB
     DEBUG_uart_print_array_hex(rx_buffer, 5);
-    DEBUG_uart_print_str("\n");
+    DEBUG_uart_print_str("\t");
```

Comment:
Ideally, use the `LOG_message` function. Avoid using `\t` control character; not certain how it's going to behave with the ground station software.

Reply:
Quick Question! Did we want this log to be part of LOG_SEVERITY_DEBUG or LOG_SEVERITY_NORMAL, since these messages were not setup to be part of the telecommand response buffer? Currently the is_reachable telecommand only prints the chip# availability and error codes if any. If the capacities are important we can have them printed part of the response itself.

Reply:
Probably NORMAL